README.md

```diff
@@ -19,21 +21,21 @@ This can be used alongside other plugins such as [@hicksy/arc-plugin-sandbox-str
 npm install -D @nasa-gcn/architect-plugin-dynamodb-local
 ```
 
-1. In your `.env` add the following:
+2. In your `.env` add the following:
 
 ```
 ARC_TABLES_PORT=8000
 ARC_DB_EXTERNAL=true
 ```
 
-1. Add the following to your project's `app.arc` configuration file:
+3. Add the following to your project's `app.arc` configuration file:
 
 ```
 @plugins
 nasa-gcn/architect-plugin-dynamodb-local
 ```
 
-1. Seeding the database (optional):
+4. Seeding the database (optional):
```

Comment:
Would you please fix the item numbers on the main branch?

Reply:
I'd rather just keep this all in one PR. I don't see the benefit in making another when I am already making changes to this file here too

Reply:
Normally we want a commit or a PR to be a self-contained set of related changes. This is important because a clean commit history helps whenever we need to perform serious analysis of past commits --- e.g., when looking for the commit that introduced a regression, especially when using `git bisect`.

Is there a reason why this typographical change must be in the same PR as the table streams code?

Reply:
Yes, I noticed it while making a change to the other change to this markdown file. And while I agree that distinct changes is beneficial for a commit history, making another PR only to have "typo change" as part of a clean history is pedantic. 

I will revert this change and make a separate PR, but I disagree with the rationale

Reply:
This is small enough a change that you can just push the renumbering onto the main branch if you like.